src/index.js

```diff
@@ -64,15 +64,22 @@ window.wikiaJWPlayer = function (elementId, options, callback) {
 				image: '//content.jwplatform.com/thumbs/' + videoId + '-640.jpg',
 				mute: options.mute,
 				playlist: options.videoDetails.playlist,
-				title: options.videoDetails.title
+				title: options.videoDetails.title,
+				captions: {
```

Comment:
Shouldn't it be override by styles property if provided?

Reply:
it's not inlined so it's overridden by css anyway, that's why I'll aim to style them via `::cue` in css.